files/N+1/data.tex

```diff
@@ -0,0 +1,22 @@
+\section{Data persistence}
+This section describes how data persistence is obtained and also give insight into the database schema shown on figure \ref{dbSchema}
+
+To persist data this project uses \glsi{mongodb} which is a \gls{documentdb}.
+When working with \glspl{documentdb} you often get a more flat and embedded data model than when working with relational databases. 
+This is also reflected in our database schema on figure \ref{dbSchema} which only has 2 entities which will translate into 2 collections in the \glsi{mongodb}.
+
+A node environment is used on the server to communicate with the \gls{mongodb} through a npm \gls{mongodb} driver.
+
+\subsection{Database schema}
+After analyzing the domain and our requirements the following database schema was created to fulfill our needs.
+
+It contains 2 collections i.e. Measurement and User. Users represent all possible users of the system, both players, consultants and coaches. 
+You then distinguish between then from flags on the document (isConsultant and isCoach).
```

Comment:
Rephrase to 'They are' instead:
The are then distinguished with flags on the document.
'isConsultant' and 'isCoach' should be in \verb++